docs/en/Xamarin.Essentials/NotImplementedInReferenceAssemblyException.xml

```diff
@@ -11,7 +11,7 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>Exception that occurs when executed from a platform that is not supported.</summary>
+    <summary>Exception that occurs when executed from a reference assembly. This usually means that the NuGet was not installed into the app project.</summary>
```

Comment:
Re-wording as this exception should only ever be thrown from reference assemblies. If this is thrown in normal code, then it is a bug.